SoftLayer/CLI/block/list.py

```diff
@@ -0,0 +1,66 @@
+"""List block storage volumes."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import columns as column_helper
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+COLUMNS = [
+    column_helper.Column('id', ('id',), mask="id"),
+    column_helper.Column('username', ('username',), mask="username"),
+    column_helper.Column('datacenter',
+                         ('serviceResource', 'datacenter', 'name'),
+                         mask="serviceResource.datacenter.name"),
+    column_helper.Column(
+        'storageType',
+        lambda b: b['storageType']['keyName'].split('_').pop(0),
```

Comment:
TODO: check if storageType.keyName is okay to show to users... this just seems like a hack and is repeated in multiple places